packages/now-python/src/pipInstall.ts

```diff
@@ -0,0 +1,120 @@
+import execa from 'execa';
+import { debug, Meta } from '@now/build-utils';
+const pipPath = 'pip3';
+
+async function isInstalled(dependency: string, cwd: string) {
+  try {
+    await execa('python3', ['-c', `"import ${dependency}"`], {
+      stdio: 'pipe',
+      cwd,
+    });
+    return true;
+  } catch (err) {
+    return false;
+  }
+}
+
+const makeRequirementsCheckCode = (requirementsPath: string) => `
+import distutils.text_file
+import pkg_resources
+from pkg_resources import DistributionNotFound, VersionConflict
+dependencies = distutils.text_file.TextFile(filename='${requirementsPath}').readlines()
+pkg_resources.require(dependencies)
```

Comment:
What happens when there is a line in `requirements.txt` that is empty or a comment? Will this still work?

Reply:
it will work, it parses the file as `pip install -r` does